config/locales/en.yml

```diff
@@ -64,6 +64,7 @@ en:
     add_all: 'Add all'
     select_all: 'Select all'
     remove: 'Remove'
+    report: 'Report'
```

Comment:
Remember that every added key in en.yml needs to be translated into other languages. Feel free to use Google Translate for now.